src/vscode/superbol-vscode-platform/superbol_languageclient.ml

```diff
@@ -48,7 +48,8 @@ let find_superbol root =
     Vscode.Uri.joinPath root ~pathSegments:[name]) @@ [
     Format.asprintf "%s-%s-%s%s" prefix platform arch suffix;
     Format.asprintf "%s-%s%s" prefix platform suffix;
-    Format.asprintf "%s%s" prefix suffix
+    Format.asprintf "%s%s" prefix suffix;
+    Format.asprintf "main.bc.js"
```

Comment:
Here that comes before the `arm64`->`x64` special case. I guess we'd need that further down the list.

Reply:
Good catch, I'll add it after the last check